### PR TITLE
[Doc] added description of monitoring setting write_direct

### PR DESCRIPTION
**Pull request by andsel**
Describe the setting `xpack.monitoring.collection.write_direct.enabled` introduced with PR #11541. The behaviour of this flag is to enable the internal collectors to shape the metrics documents in a way to skip the Elasticsearch's monitoring exporter. Essentially we do the same job the exporter already does, and we switch the endpoint we use to push data from `_monitoring/bulk` to `_bulk/monitoring`

### CODE REVIEW

**Comment by andsel** [comment]
I've added, but in this other PR #11541

File: docs/static/settings/monitoring-settings.asciidoc [hunk 1]
```diff
@@ -50,6 +50,11 @@ Controls how often data samples are collected and shipped on the Logstash side.
 Defaults to `10s`. If you modify the collection interval, set the 
 `xpack.monitoring.min_interval_seconds` option in `kibana.yml` to the same value.
 
+`xpack.monitoring.collection.write_direct.enabled`::
+
+Forces the internal collectors to write directly to the monitoring {es} without
+traversing the monitoring exporter provided by {es}. Disabled by default, this setting
+is intermediate step in deprecating the internal collectors.
```
Comment (robbavey) [inline docs/static/settings/monitoring-settings.asciidoc]: Adding to my [comment](https://github.com/elastic/logstash/pull/11541#discussion_r376105959) with the code changes, I suspect we are also going to need to do some more extensive documentation changes for this update to mirror the [docs](https://www.elastic.co/guide/en/beats/filebeat/current/monitoring.html) for filebeat monitoring, where there are three options for monitoring: Legacy collection (monitoring data sent via production cluster), Internal collection (monitoring data sent direct to monitoring cluster) and metricbeat collection.

Comment (andsel) [reply docs/static/settings/monitoring-settings.asciidoc]: I've changed a little bit the documenatiton part related to xpack monitoring, let me know what do you think @karenzone

**Comment by robbavey** [comment]
@karenzone If you are willing to do that, that would be awesome. I think this PR is a good jumping off point, but there are a few areas in the (new) internal collection section that could do with a bit of clarification/clean up - the docs still reference sending to the production cluster, and it might be nice to talk about using the new `cloud.id` settings to point to a cloud monitoring cluster as well as tweak a few other areas that might be confusing (how to set `cluster_uuid`, etc)

**Comment by andsel** [comment]
I agree with you

**Comment by andsel** [comment]
Close this because work has been moved into #11714 and #11789

**Pull request closed without merging**